pages/content/amp-dev/community/roadmap.html

```diff
@@ -2,86 +2,185 @@
+    <div class="ap-t-roadmap-navigation">
+      <div class="ap-t-roadmap-navigation-list">
+        {% for quarter in doc.roadmap.list|map(attribute='quarter')|unique|list %}
+
+        {% set roadmap_quarter = dict(doc.roadmap.list|groupby('quarter')) %}
+        [% set roadmap_filtered = {{ dict(roadmap_quarter[quarter]|groupby('wg_slug'))|tojson }} %]
+
+        [% if not category or roadmap_filtered[category]|length > 0 %]
+        <amp-animation id="anim-{{ quarter|slug }}" layout="nodisplay">
+          <script type="application/json">
+            {
+              "duration": ".3s",
+              "fill": "both",
+              "direction": "alternate",
+              "animations": [
+                {
+                  "selector": ".ap-t-roadmap-navigation-list-item",
+                  "keyframes": {
+                    "opacity": ".3"
+                  }
+                },
+                {
+                  "selector": "#nav-item-{{ quarter|slug }}",
+                  "keyframes": {
+                    "opacity": "1"
+                  }
+                },
+                {
+                  "selector": ".ap-t-roadmap-navigation-list-item-timeline",
+                  "keyframes": {
+                    "opacity": "0"
+                  }
+                },
+                {
+                  "selector": "#nav-item-timeline-{{ quarter|slug }}",
+                  "keyframes": {
+                    "opacity": "1"
+                  }
+                }
+              ]
+            }
+          </script>
+        </amp-animation>
+        <button class="ap-t-roadmap-navigation-list-item"
+            id="nav-item-{{ quarter|slug }}"
+            on="tap:{{ quarter|slug }}.scrollTo(duration=250)">
+          <h3 class="ap-t-roadmap-navigation-list-item-title">
+            {{ quarter }}
+          </h3>
+          <span class="ap-t-roadmap-navigation-list-item-timeline"
+              id="nav-item-timeline-{{ quarter|slug }}">
+          </span>
+        </button>
+        [% endif %]
+
+        {% endfor %}
+      </div>
+    </div>
+
+    [% set quarters = {{ doc.roadmap.quarters|tojson }} %]
+    <div class="ap-t-roadmap-list">
+      {% do doc.icons.useIcon('icons/chevron-vertical-light.svg') %}
+
+      {% for issue in doc.roadmap.list %}
+      {% if loop.changed(issue.quarter) %}
+      [% if not category or quarters['{{ issue.quarter }}'].includes(category) %]
+      <amp-position-observer viewport-margins="25vh"
+          target="{{ issue.quarter|slug }}"
+          on="enter:anim-{{ issue.quarter|slug }}.start"
+          layout="nodisplay">
+      </amp-position-observer>
+      <div class="ap-t-roadmap-list-quarter-separator" id="{{ issue.quarter|slug }}">
+        <h3>{{ issue.quarter }}</h3>
+      </div>
+      [% endif %]
+      {% endif %}
+
+      [% if category in '{{ issue.wg_slug }}' %]
```

Comment:
I stumbled over this earlier and you've been telling me that `==` didn't work, now I know what's wrong: `category` can be `undefined` if no category is selected. This `undefined in 'wg-name'` seems to evaluate to `true` in nunjucks. What you actually want is:

```
[% if not category or category == '{{ issue.wg_slug }}' %]
```